temporal-sdk/src/test/java/io/temporal/internal/testing/ActivityTestingTest.java

```diff
@@ -111,6 +109,21 @@ public void testFailure() {
     }
   }
 
+  private static class AsyncActivityImpl implements TestActivity {
+    @Override
+    public String activity1(String input) {
+      Activity.getExecutionContext().doNotCompleteOnReturn();
```

Comment:
When running in a non-test-environment, does this throw? If not, not sure we should in test environment either. If it does, can we throw the same thing in the test environment?

Reply:
`doNotCompleteOnReturn ` behaves the same in the test environment and non-test-environment